sgl-kernel/src/sgl-kernel/cpu.py

```diff
@@ -0,0 +1,86 @@
+import sgl_kernel
+import torch
+
+
+def fused_experts(
+    x,
+    w13_weight,
+    w2_weight,
+    topk_weights,
+    topk_ids,
+    inplace,
+    is_vnni,
```

Comment:
I think we can use default parameter for `is_vnni=True`, so that you don't have to provide is_vnni when calling it.

is_vnni is mostly for debugging purpose.

Reply:
Changed to `is_vnni=True`